data/json/mutations.json

```diff
@@ -726,7 +726,18 @@
         "starting_trait" : true,
         "category" : ["PLANT"],
         "cancels" : ["GOODHEARING"],
+        "changes_to" : ["DEAF"],
         "hearing_modifier" : 0.5
+    },{
+        "type" : "mutation",
+        "id" : "DEAF",
+        "name" : "Deaf",
+        "points" : -4,
+        "description" : "Your world is in complete slience. You can't talk to NPC's",
```

Comment:
```suggestion
        "description" : "Your world is in complete silence.  You can't talk to NPCs.",
```

Reply:
Adding onto this, I feel like `You can't hear anything, and can't talk to NPCs.` would flow a bit better.